compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -70,6 +70,9 @@ int32_t get_broadcastTo_index_among_inputs_of(luci::CircleCustom *cop)
   return -1;
 }
 
+/// @note  When adding two input values of different shapes, Add performs broadcasting.
```

Comment:
This PR comes form the discussion here. ---> https://github.com/Samsung/ONE/pull/12509#discussion_r1480662054